pandas/tseries/tests/test_resample.py

```diff
@@ -741,15 +741,15 @@ def test_resample_empty_dataframe(self):
 
         for freq in ['M', 'D', 'H']:
             # count retains dimensions too
-            methods = downsample_methods + ['count']
+            methods = downsample_methods + upsample_methods
             for method in methods:
                 result = getattr(f.resample(freq), method)()
 
-                expected = f.copy()
+                expected = pd.Series([])
```

Comment:
@jreback. Could you please clarify the name you talking about? The name of the `Series`? How it could exist if the `Series` created after DataFrame resampling.